bin/inx-chronicle/src/api/config.rs

```diff
@@ -13,17 +17,63 @@ use super::error::ConfigError;
 pub struct ApiConfig {
     pub port: u16,
     pub allow_origins: Option<SingleOrMultiple<String>>,
+    pub password_hash: String,
+    pub password_salt: String,
+    pub jwt_salt: String,
+    #[serde(with = "humantime_serde")]
+    pub jwt_expiration: Duration,
+    pub public_routes: Vec<String>,
 }
 
 impl Default for ApiConfig {
     fn default() -> Self {
         Self {
             port: 9092,
-            allow_origins: Some(String::from("*").into()),
+            allow_origins: Some("*".to_string().into()),
+            password_hash: "c42cf2be3a442a29d8cd827a27099b0c".to_string(),
+            password_salt: "saltines".to_string(),
+            jwt_salt: "Chronicle".to_string(),
+            // 30 min
+            jwt_expiration: Duration::from_secs(30 * 60),
```

Comment:
With the changes to the `config.template.toml`, we can make the expiration optional and set it to `None` by default.